packages/mui-base/src/NumberField/NumberFieldScrubArea.test.tsx

```diff
@@ -47,16 +54,11 @@ describe('<NumberField.ScrubArea />', () => {
     expect(screen.queryByRole('presentation')).not.to.equal(null);
   });
 
-  if (/jsdom/.test(window.navigator.userAgent)) {
+  // Only run the following tests in Chromium/Firefox.
+  if (/jsdom/.test(window.navigator.userAgent) || isWebKit()) {
     return;
   }
```

Comment:
Not sure why Safari 15.x is failing here, but in my tests manually in the latest version it works correctly. Possibly something to do with the manual dispatching of events, as it may actually work in Safari 15.